flutter_local_notifications/lib/src/platform_specifics/darwin/notification_details.dart

```diff
@@ -139,4 +140,16 @@ class DarwinNotificationDetails {
   /// This property is only applicable to iOS 15.0 and macOS 12.0 or newer.
   /// https://developer.apple.com/documentation/usernotifications/unnotificationcontent/3747256-interruptionlevel
   final InterruptionLevel? interruptionLevel;
+
+  /// The volume for playing the critical alert sound.
+  ///
+  /// The value is between 0.0 and 1.0.
+  /// This is only used when the notification is configured as a critical alert.
+  /// Document: https://developer.apple.com/documentation/usernotifications/unnotificationsound/criticalsoundnamed(_:withaudiovolume:)
+  ///
+  /// Subject to specific approval from Apple:
+  /// https://developer.apple.com/contact/request/notifications-critical-alerts-entitlement/
+  ///
+  /// This property is only applicable to iOS 12.0 or newer.
```

Comment:
I believe this is missing mention of how it's also only applicable macOS 10.14 or newer too? If so, see some of the existing API docs in the code on the convention being used when a property is specific to certain versions of iOS and macOS. For convenience it's in the format of

```
On iOS, this property is only applicable to iOS <...> or newer.
On macOS, this property is only applicable to macOS <...> or newer.
```